### DISCUSSION TITLE
Kernel Panic on Install

### DISCUSSION THREAD

**Discussion by 1689er** [Security-Onion-Solutions/securityonion#11013] [General] [answered]
Hi all. I'm trying to install 2.3.260-20230620 from bootable USB drive. I've tested the media and it's good, I've also tested the hardware and it's good. It doesn't seem to matter which Install option I choose on the Install page they all give me the error. I've put a picture below. Any ideas where to go from where? I'm trying to install on an ASRock Industrial NUC BOX-1260P (https://www.newegg.com/asrock-nuc-box-1260p/p/N82E16856158081?Item=N82E16856158081). 
![PXL_20230810_203922808](https://github.com/Security-Onion-Solutions/securityonion/assets/4614923/9c60236b-695a-410d-ad7e-78069061d49f)

**Comment by dougburks** [comment, marked as answer]
You can try one of the alternate 2.3 installation methods here:
https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos

OR you can try the new 2.4 Release Candidate 2:
https://blog.securityonion.net/2023/08/security-onion-24-release-candidate-2.html

**Reply by 1689er** [reply]
Thank you Doug. The new 2.4 release worked! I guess something in 2.3 didn't like my setup or something, IDK. :)
